core/controllers/base.py

```diff
@@ -200,8 +222,6 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
                 # the not-fully registered user.
                 email = auth_claims.email
                 if 'signup?' in self.request.uri:
-                    if not feconf.ENABLE_USER_CREATION:
```

Comment:
Where is the replacement check for this?

Reply:
Unneeded, maintenance mode already handles returning early on line 214 (so adding an equivalent test/check would hit unreachable code).